tools/install.sh

```diff
@@ -122,6 +123,26 @@ else
     ANKAIOS_RELEASE_URL_SHA="${RELEASE_URL_BASE}/download/${ANKAIOS_VERSION}/${RELEASE_FILE_NAME_WITH_SHA}"
 fi
 
+if [ -z "$INSTALL_ANK_SERVER_RUST_LOG" ] ; then
+    echo "No log level for ank-server provided, use default: 'info'"
+    INSTALL_ANK_SERVER_RUST_LOG=${DEFAULT_LOG_LEVEL}
+else
+    if [[ ! "${INSTALL_ANK_SERVER_RUST_LOG}" =~ ^(info|debug|warn|error|trace)$ ]]; then
```

Comment:
Why should we restrict the content of the log level? I think the user should have all freedom including settings different log levels per component (see [documentation](https://docs.rs/env_logger/latest/env_logger/#enabling-logging)). So we should accept all values and just pass them to `RUST_LOG` with quotes.

Reply:
Done.